tools/python/transit/gtfs/download_gtfs.py

```diff
@@ -45,13 +46,17 @@ def download_gtfs_sources_mobilitydb(path):
             f"HTTP error {http_err} downloading zip from {URL_MOBILITYDB_GTFS_SOURCE}")
 
 
-def get_gtfs_urls_mobilitydb(path):
+def get_gtfs_urls_mobilitydb(path, countries_list):
     """Extracts the feed urls from the downloaded csv file"""
+    download_from_all_countries = True
+    if countries_list:
+        download_from_all_countries = False
+
     download_gtfs_sources_mobilitydb(path)
     file = open(os.path.join(path, RAW_FILE_MOBILITYDB), encoding='UTF-8')
     raw_sources = csv.DictReader(file)
     next(raw_sources)
-    urls = [field["urls.direct_download"] for field in raw_sources]
+    urls = [field["urls.direct_download"] for field in raw_sources if field["location.country_code"] in countries_list or download_from_all_countries]
```

Comment:
nit: I would also swap the OR check to make it simpler:
`if download_from_all_countries or field["location.country_code"] in countries_list`

Do you see the reasoning? :)